src/services/userServices.js

```diff
@@ -20,3 +20,10 @@ export const updateUserInfo = credentials => {
     .then(res => Promise.resolve(res.data))
     .catch(err => Promise.reject(err.response.data.error));
 };
+
+export const validateToken = userId => {
+  return Api()
+    .get(`api/v1/user/validateToken/${userId.id}`)
```

Comment:
Oh, what I meant is that there shouldn't be an object property location here again in the actual service. It should just be `userId`.

There could be `anything.anything` in the place this function is now used though.